packages/core/src/domain/session/sessionState.ts

```diff
@@ -1,31 +1,58 @@
 import { isEmptyObject } from '../../tools/utils/objectUtils'
 import { objectEntries } from '../../tools/utils/polyfills'
 import { dateNow } from '../../tools/utils/timeUtils'
-import { SESSION_EXPIRATION_DELAY } from './sessionConstants'
+import { SESSION_EXPIRATION_DELAY, SESSION_TIME_OUT_DELAY } from './sessionConstants'
 
-const SESSION_ENTRY_REGEXP = /^([a-z]+)=([a-z0-9-]+)$/
+const SESSION_ENTRY_REGEXP = /^([a-zA-Z]+)=([a-z0-9-]+)$/
 const SESSION_ENTRY_SEPARATOR = '&'
 
+export const EXPIRED = '1'
+
 export interface SessionState {
   id?: string
   created?: string
   expire?: string
   lock?: string
+  isExpired?: typeof EXPIRED
 
   [key: string]: string | undefined
 }
 
-export function isSessionInExpiredState(session: SessionState) {
+export function getExpiredSessionState(): SessionState {
+  return {
+    isExpired: EXPIRED,
+  }
+}
+
+export function isSessionInNotStartedState(session: SessionState) {
   return isEmptyObject(session)
 }
 
+export function isSessionStarted(session: SessionState) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏To be consistent other state functions 
```suggestion
export function isSessionInStartedState(session: SessionState)
```

Reply:
yeah, although as I mentioned in the doc, `started` is not really a state, it is either one of the following states: `Expired`, `Tracked`, or `NotTracked`.

Reply:
It still seems strange to me that "NotStarted" is a state but "Started" is not. Basically having only negated state confuses me, but I won't block the PR for that. This is clearly a tough one 🙂